provision/pmc/templates/client-mu-plugins-plugin-loader.php

```diff
@@ -22,3 +22,18 @@
 
 	unset( $theme_plugins_path );
 } );
+
+/**
+ * We use wpcom_vip_load_plugin() in contexts that occur after the
+ * `plugins_loaded` action has already fired, which triggers a
+ * _doing_it_wrong() message in our logs. We don't plan on changing our
+ * approach here, so mute those notices.
+ */
+function pmc_mute_wpcom_vip_load_plugin_notice( $trigger, $function, $message ) {
+	$needle = 'as called after the `plugins_loaded` hook. For best results, we recommend loading your plugins earlier from `client-mu-plugins';
```

Comment:
To future-proof this against VIP changing the message, let's try this:

```
		if ( 'wpcom_vip_load_plugin' !== $function ) {
			return $do_error;
		}

		if ( false === strpos( $message, 'plugins_loaded' ) ) {
			return $do_error;
		}

		return false;
```

Reply:
Thanks, for some reason `$function` didn't seem to work in my original testing. Updated.